core/embed/gfx/fonts/fonts.h

```diff
@@ -21,7 +21,7 @@
 #define _FONTS_H
 
 #include <stdbool.h>
-
+#include <stdint.h>
```

Comment:
Instead of including standard headers, we now include `trezor_types.h` in API files and `trezor_rtl.h` in implementation files. Therefore, this should be `#include <trezor_types.h>`. But in general there's nothing wrong on using `stdint.h` in  generated files like `font_unifont_bold_16.h`.

Reply:
https://github.com/trezor/trezor-firmware/commit/d42cea3818e154945ac7649bb2031822192adddb